packages/phalanx/test/Kokkos/tKokkosViewOfViews.cpp

```diff
@@ -0,0 +1,60 @@
+#include "Kokkos_Core.hpp"
+#include "Teuchos_UnitTestHarness.hpp"
+
+// This test demonstrates how to create a view of views for double and FAD types.
+using exec_t = Kokkos::DefaultExecutionSpace;
+using mem_t = Kokkos::DefaultExecutionSpace::memory_space;
+
+TEUCHOS_UNIT_TEST(ViewOfViews,double) {
+
+  const int num_cells = 10;
+  const int num_pts = 8;
+  const int num_equations = 32;
+
+  Kokkos::View<double***,mem_t> a("a",num_cells,num_pts,num_equations);
+  Kokkos::View<double***,mem_t> b("b",num_cells,num_pts,num_equations);
+  Kokkos::View<double***,mem_t> c("c",num_cells,num_pts,num_equations);
+
```

Comment:
Do you want to use PHX::View here??

Reply:
This directory is to verify that kokkos code works for phalanx use cases before I screw things up with phalanx code. For now, keeping it phalanx free. I'll add another test with phalanx data structures soon.

Reply:
You don't need one, but just cheking,